oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -142,6 +142,12 @@ configuration:
 #    period : 5
 #    # the name of current cluster, set the name if you want to upstream system known.
 #    clusterName: "default"
+  zookeeper:
```

Comment:
should be commented out by default

Reply:
What is the reason it?

Reply:
Yes. @kezhenxu94 is right. DCS should keep off in default because we don't know user even has a config center.